test/util/mvc/controllers/plainHTMLController.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (app) => {
+  // route to request the plain HTML page and return the HTML page
+  app.route('/HTMLTest').get((req, res) => {
+    // render the teddy template and pass it the model
+    res.send('plainHTMLTest')
```

Comment:
For a `res.send` you need to read the HTML file itself. The best way to do this would be caching a variable:

(require `fs` and `path` outside of the module.exports)

`const plainHTMLPage = fs.readFileSync(path.join(app.get('viewsPath'), plainHTMLTest.html))`